libr/parse/parse.c

```diff
@@ -262,14 +262,22 @@ static int filter(RParse *p, ut64 addr, RFlag *f, char *data, char *str, int len
 			}
 			if (f) {
 				RFlagItem *flag2;
-				flag = r_flag_get_i2 (f, off);
+				flag = r_flag_get_by_spaces (f, off, 4, "symbols", "imports",
+						"relocs", "strings");
+				if (!flag) {
+					flag = r_flag_get_i2 (f, off);
```

Comment:
i believe this shouldn't be needed anymore if `r_flag_get_by_spaces` works well, should it?

Reply:
what about this? are there reasons to keep it?